Linux compile failure due to missing #include: rpcs3/Emu/RSX/VK/vkutils/physical_device.h

Tried to build 0.0.14 today and finding that it is failing on both gcc (10.2.0) and clang (11.0.0).  It would appear that `rpcs3/Emu/RSX/VK/vkutils/physical_device.h` is missing `#include <string>`.  Patching to add this include fixes the build.

Commit hash tested:  https://github.com/RPCS3/rpcs3/commit/5f8564c6558958dc5626a26004bd0af3a697d5b3

Patch with fix:

```
diff --git a/rpcs3/Emu/RSX/VK/vkutils/physical_device.h b/rpcs3/Emu/RSX/VK/vkutils/physical_device.h
index 6885006cc..c8e708daf 100644
--- a/rpcs3/Emu/RSX/VK/vkutils/physical_device.h
+++ b/rpcs3/Emu/RSX/VK/vkutils/physical_device.h
@@ -6,6 +6,7 @@
 
 #include <vector>
 #include <unordered_map>
+#include <string>
 
 namespace vk
 {
```

gcc log:
```
[1857/1997] /usr/bin/x86_64-pc-linux-gnu-g++ -DGLX_GLXEXT_PROTOTYPES -DGL_GLEXT_PROTOTYPES -DHAVE_ALSA -DHAVE_LIBEVDEV -DHAVE_PULSE -DHAVE_VULKAN -DHAVE_X11 -DLLVM_AVAILABLE -DNDEBUG -DWC_NO_HARDEN -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3 -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5 -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/yaml-cpp/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/flatbuffers/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/pugixml/src -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/span/include -I/usr/include/AL -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/stblib -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/llvm/include -I3rdparty/llvm_build/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/asmjit/src -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/cereal/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/GL -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/Vulkan/glslang/SPIRV/.. -Iinclude -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/Vulkan/spirv-tools/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/Vulkan/spirv-headers/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/libusb_cmake/../libusb/libusb -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/wolfssl -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/xxHash/cmake_unofficial/.. -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/xxHash -isystem /usr/include/libevdev-1.0  -march=native -O3 -fgraphite-identity -floop-nest-optimize -fdevirtualize-at-ltrans -fipa-pta -fno-semantic-interposition -flto=32 -fuse-linker-plugin -pipe -Wl,-O1 -Wl,--as-needed -Wl,--exclude-libs,ALL -pthread -Wall -fno-exceptions -ftemplate-depth=1024 -msse -msse2 -mcx16 -Werror=old-style-cast -Werror=sign-compare -Werror=reorder -Wno-return-type -Wno-unused-variable -Wno-unknown-pragmas -Wno-invalid-offsetof -Wno-unused-function -Wno-attributes -Wno-comment -Wno-maybe-uninitialized -Wno-strict-aliasing -Wno-unused-but-set-variable -Wno-class-memaccess -std=gnu++2a -MD -MT rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o -MF rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o.d -o rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o -c /var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp
FAILED: rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o 
/usr/bin/x86_64-pc-linux-gnu-g++ -DGLX_GLXEXT_PROTOTYPES -DGL_GLEXT_PROTOTYPES -DHAVE_ALSA -DHAVE_LIBEVDEV -DHAVE_PULSE -DHAVE_VULKAN -DHAVE_X11 -DLLVM_AVAILABLE -DNDEBUG -DWC_NO_HARDEN -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3 -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5 -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/yaml-cpp/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/flatbuffers/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/pugixml/src -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/span/include -I/usr/include/AL -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/stblib -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/llvm/include -I3rdparty/llvm_build/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/asmjit/src -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/cereal/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/GL -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/Vulkan/glslang/SPIRV/.. -Iinclude -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/Vulkan/spirv-tools/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/Vulkan/spirv-headers/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/libusb_cmake/../libusb/libusb -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/wolfssl -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/xxHash/cmake_unofficial/.. -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/xxHash -isystem /usr/include/libevdev-1.0  -march=native -O3 -fgraphite-identity -floop-nest-optimize -fdevirtualize-at-ltrans -fipa-pta -fno-semantic-interposition -flto=32 -fuse-linker-plugin -pipe -Wl,-O1 -Wl,--as-needed -Wl,--exclude-libs,ALL -pthread -Wall -fno-exceptions -ftemplate-depth=1024 -msse -msse2 -mcx16 -Werror=old-style-cast -Werror=sign-compare -Werror=reorder -Wno-return-type -Wno-unused-variable -Wno-unknown-pragmas -Wno-invalid-offsetof -Wno-unused-function -Wno-attributes -Wno-comment -Wno-maybe-uninitialized -Wno-strict-aliasing -Wno-unused-but-set-variable -Wno-class-memaccess -std=gnu++2a -MD -MT rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o -MF rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o.d -o rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o -c /var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp
In file included from /var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp:1:
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.h:61:8: error: ‘string’ in namespace ‘std’ does not name a type
   61 |   std::string get_name() const;
      |        ^~~~~~
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.h:8:1: note: ‘std::string’ is defined in header ‘<string>’; did you forget to ‘#include <string>’?
    7 | #include <vector>
  +++ |+#include <string>
    8 | #include <unordered_map>
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.h:64:8: error: ‘string’ in namespace ‘std’ does not name a type
   64 |   std::string get_driver_version() const;
      |        ^~~~~~
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.h:64:3: note: ‘std::string’ is defined in header ‘<string>’; did you forget to ‘#include <string>’?
   64 |   std::string get_driver_version() const;
      |   ^~~
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp: In member function ‘void vk::physical_device::create(VkInstance, VkPhysicalDevice, bool)’:
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp:67:76: error: ‘get_name’ was not declared in this scope; did you mean ‘rename’?
   67 |   rsx_log.always("Found vulkan-compatible GPU: '%s' running on driver %s", get_name(), get_driver_version());
      |                                                                            ^~~~~~~~
      |                                                                            rename
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp:67:88: error: ‘get_driver_version’ was not declared in this scope; did you mean ‘get_driver_vendor’?
   67 |   rsx_log.always("Found vulkan-compatible GPU: '%s' running on driver %s", get_name(), get_driver_version());
      |                                                                                        ^~~~~~~~~~~~~~~~~~
      |                                                                                        get_driver_vendor
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp: At global scope:
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp:98:14: error: no declaration matches ‘std::string vk::physical_device::get_name() const’
   98 |  std::string physical_device::get_name() const
      |              ^~~~~~~~~~~~~~~
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp:98:14: note: no functions named ‘std::string vk::physical_device::get_name() const’
In file included from /var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp:1:
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.h:33:8: note: ‘class vk::physical_device’ defined here
   33 |  class physical_device
      |        ^~~~~~~~~~~~~~~
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp: In member function ‘vk::driver_vendor vk::physical_device::get_driver_vendor() const’:
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp:107:26: error: ‘get_name’ was not declared in this scope; did you mean ‘gpu_name’?
  107 |    const auto gpu_name = get_name();
      |                          ^~~~~~~~
      |                          gpu_name
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp: At global scope:
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp:151:14: error: no declaration matches ‘std::string vk::physical_device::get_driver_version() const’
  151 |  std::string physical_device::get_driver_version() const
      |              ^~~~~~~~~~~~~~~
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp:151:14: note: no functions named ‘std::string vk::physical_device::get_driver_version() const’
In file included from /var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp:1:
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.h:33:8: note: ‘class vk::physical_device’ defined here
   33 |  class physical_device
      |        ^~~~~~~~~~~~~~~
```

clang log:
```
[1863/1997] /usr/lib/llvm/11/bin/clang++ -DGLX_GLXEXT_PROTOTYPES -DGL_GLEXT_PROTOTYPES -DHAVE_ALSA -DHAVE_LIBEVDEV -DHAVE_PULSE -DHAVE_VULKAN -DHAVE_X11 -DLLVM_AVAILABLE -DNDEBUG -DWC_NO_HARDEN -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3 -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5 -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/yaml-cpp/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/flatbuffers/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/pugixml/src -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/span/include -I/usr/include/AL -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/stblib -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/llvm/include -I3rdparty/llvm_build/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/asmjit/src -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/cereal/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/GL -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/Vulkan/glslang/SPIRV/.. -Iinclude -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/Vulkan/spirv-tools/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/Vulkan/spirv-headers/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/libusb_cmake/../libusb/libusb -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/wolfssl -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/xxHash/cmake_unofficial/.. -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/xxHash -isystem /usr/include/libevdev-1.0  -march=native -O3 -pipe -Wl,--exclude-libs,ALL -pthread -Wall -fno-exceptions -ftemplate-depth=1024 -msse -msse2 -mcx16 -Werror=old-style-cast -Werror=sign-compare -Werror=reorder -Wno-return-type -Wno-unused-variable -Wno-unknown-pragmas -Wno-invalid-offsetof -Wno-unused-function -Wno-attributes -Wno-comment -fconstexpr-steps=16777216 -Wno-sometimes-uninitialized -Wno-unused-lambda-capture -Wno-unused-private-field -Wno-self-assign -Wno-pessimizing-move -Wno-delete-non-virtual-dtor -Wno-unused-command-line-argument -std=gnu++20 -MD -MT rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o -MF rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o.d -o rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o -c /var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp
FAILED: rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o 
/usr/lib/llvm/11/bin/clang++ -DGLX_GLXEXT_PROTOTYPES -DGL_GLEXT_PROTOTYPES -DHAVE_ALSA -DHAVE_LIBEVDEV -DHAVE_PULSE -DHAVE_VULKAN -DHAVE_X11 -DLLVM_AVAILABLE -DNDEBUG -DWC_NO_HARDEN -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3 -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5 -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/yaml-cpp/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/flatbuffers/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/pugixml/src -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/span/include -I/usr/include/AL -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/stblib -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/llvm/include -I3rdparty/llvm_build/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/asmjit/src -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/cereal/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/GL -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/Vulkan/glslang/SPIRV/.. -Iinclude -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/Vulkan/spirv-tools/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/Vulkan/spirv-headers/include -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/libusb_cmake/../libusb/libusb -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/wolfssl -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/xxHash/cmake_unofficial/.. -I/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/3rdparty/xxHash -isystem /usr/include/libevdev-1.0  -march=native -O3 -pipe -Wl,--exclude-libs,ALL -pthread -Wall -fno-exceptions -ftemplate-depth=1024 -msse -msse2 -mcx16 -Werror=old-style-cast -Werror=sign-compare -Werror=reorder -Wno-return-type -Wno-unused-variable -Wno-unknown-pragmas -Wno-invalid-offsetof -Wno-unused-function -Wno-attributes -Wno-comment -fconstexpr-steps=16777216 -Wno-sometimes-uninitialized -Wno-unused-lambda-capture -Wno-unused-private-field -Wno-self-assign -Wno-pessimizing-move -Wno-delete-non-virtual-dtor -Wno-unused-command-line-argument -std=gnu++20 -MD -MT rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o -MF rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o.d -o rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/RSX/VK/vkutils/physical_device.cpp.o -c /var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp
In file included from /var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.cpp:1:
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.h:61:8: error: no type named 'string' in namespace 'std'
                std::string get_name() const;
                ~~~~~^
/var/tmp/portage/games-emulation/rpcs3-0.0.14_p5/work/rpcs3-0.0.14_p5/rpcs3/Emu/RSX/VK/vkutils/physical_device.h:64:8: error: no type named 'string' in namespace 'std'
                std::string get_driver_version() const;
                ~~~~~^
2 errors generated.
```

--- a/rpcs3/Emu/RSX/VK/vkutils/physical_device.h
+++ b/rpcs3/Emu/RSX/VK/vkutils/physical_device.h
@@ -4,6 +4,7 @@
 #include "chip_class.h"
 #include "pipeline_binding_table.h"
 
+#include <string>
 #include <vector>
 #include <unordered_map>
 

